[2022-08-26 19:28:22,115] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: trigger_sensor.trigger_task manual__2022-08-26T19:28:19+00:00 [queued]>
[2022-08-26 19:28:22,134] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: trigger_sensor.trigger_task manual__2022-08-26T19:28:19+00:00 [queued]>
[2022-08-26 19:28:22,135] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-26 19:28:22,137] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-26 19:28:22,137] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-26 19:28:22,163] {taskinstance.py:1397} INFO - Executing <Task(TriggerDagRunOperator): trigger_task> on 2022-08-26 19:28:19+00:00
[2022-08-26 19:28:22,175] {standard_task_runner.py:52} INFO - Started process 23275 to run task
[2022-08-26 19:28:22,184] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'trigger_sensor', 'trigger_task', 'manual__2022-08-26T19:28:19+00:00', '--job-id', '3289', '--raw', '--subdir', 'DAGS_FOLDER/trigger_dag.py', '--cfg-path', '/tmp/tmpdqv8hoks', '--error-file', '/tmp/tmpsjnbolw_']
[2022-08-26 19:28:22,185] {standard_task_runner.py:80} INFO - Job 3289: Subtask trigger_task
[2022-08-26 19:28:22,302] {task_command.py:371} INFO - Running <TaskInstance: trigger_sensor.trigger_task manual__2022-08-26T19:28:19+00:00 [running]> on host 9d5d2a77e6ba
[2022-08-26 19:28:22,433] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=trigger_sensor
AIRFLOW_CTX_TASK_ID=trigger_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-26T19:28:19+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-26T19:28:19+00:00
[2022-08-26 19:28:22,636] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=trigger_sensor, task_id=trigger_task, execution_date=20220826T192819, start_date=20220826T192822, end_date=20220826T192822
[2022-08-26 19:28:22,678] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-26 19:28:22,734] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
